src/sss_client/pam_sss_gss.c

```diff
@@ -492,6 +492,7 @@ static errno_t sss_cli_getenv(const char *variable_name, char **_value)
 {
     char *value = getenv(variable_name);
     if (value == NULL) {
+        *_value = NULL;
```

Comment:
Hi,

I'm fine with the fix but I wonder if it wouldn't  be more consistent to return `EOK` here since `NULL` is an allowed and expected result?

bye,
Sumit

Reply:
Do we need a helper function at all then?

(JFTR: regression was introduced while fixing https://bugzilla.redhat.com/show_bug.cgi?id=1977830 )

Reply:
I guess (as I don't have access to abovementioned bugzilla) the helper function was introduced to avoid TOC-TOU issue with KRB5CCNAME environment variable so it should stay. Saying that the function should IMHO return EOK if getenv returned NULL (as there is NO error in this case).